bin/permadelete_crash_data.py

```diff
@@ -82,7 +80,7 @@ def s3_delete_object(client, bucket, key):
         click.echo(f"s3: deleted {key!r}")
         return True
     except Exception:
-        logger.exception(f"ERROR: s3: when deleting {key!r}")
+        logger.exception("ERROR: s3: when deleting %r", key)
```

Comment:
While rewriting code, I went through and made sure to fix logging so that messages get interpolated by the logging client. That improves logging because it tracks the arguments.